assets/css/nav/_left_nav.scss

```diff
@@ -57,9 +61,45 @@
     font-weight: 500;
   }
 
+  &:focus {
+    border: 1px solid $color-eggplant-300;
+  }
+
   &.m-left-nav__link--active {
     background-color: $color-eggplant-100;
+    font-weight: 500;
+    color: $color-eggplant-800;
+  }
+}
+
+.m-left-nav__view {
+  @include button-icon(1.25rem);
+  display: flex;
+  align-items: center;
+  padding: 0.25rem 0.5rem;
+  box-sizing: border-box;
+  width: 100%;
+  height: 2.25rem;
+  border-radius: 0.375rem;
+  font-weight: 400;
+  @include font-body;
+  color: $color-gray-600;
+  text-decoration: none;
+  background: #ffffff;
```

Comment:
**nitpick:** it doesn't matter as much as with some of the other hex values, but you can use `$white` here.

Reply:
fixed in 995b9260